community/event_management/doctype/schedule/schedule.json

```diff
@@ -0,0 +1,63 @@
+{
+ "actions": [],
```

Comment:
I think we should call this doctype as Talk because it takes inputs related to a talk. Also, the Talk Info topic and about field should be here and not the Speaker registration doctype. Becuase what if a speaker wants to give multiple talks. Should they register twice?

Reply:
In that case, we can make a separate doctype for talk. This schedule doctype stores slots for each speaker, could be rendered via web template.

Reply:
But the slot is for the talk right? So talk info should be here.